backend/docs/README.md

```
@@ -32,37 +32,56 @@ Generated By: create-express-doc
   - Content-Type: `application/json`
   - Schema:
 
-    ```json
-    {
-  "msg": "Invalid Credentials"
-}
+        ```json
```

markdownlint-cli2 (0.20.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)